ibis/sql/sqlite/compiler.py

```diff
@@ -136,6 +136,23 @@ def _extract_quarter(t, expr):
     return sa.cast(t.translate(expr_new), sa.Integer)
 
 
+def _extract_woy(t, expr):
+    (arg,) = expr.op().args
+    sa_arg = t.translate(arg)
+    # (strftime('%j', date(MyDate, '-3 days', 'weekday 4')) - 1) / 7 + 1
+    return sa.cast(
+        (
+            sa.func.strftime(
+                '%j', sa.func.date(sa_arg, '-3 days', 'weekday 4')
+            )
+            - 1
+        )
+        / 7
+        + 1,
+        sa.INTEGER,
```

Comment:
I guess it's black who is creating this unreadable code. But you can use a variable here instead.

Reply:
ok I will improve that. thanks!